src/opentelemetry-tracing.ts

```diff
@@ -13,31 +13,49 @@
  * limitations under the License.
  */
 
-import {Attributes, SpanContext, Span, trace} from '@opentelemetry/api';
-import {Tracer} from '@opentelemetry/tracing';
+import {
+  Tracer,
+  SpanAttributes,
+  SpanContext,
+  Span,
+  context,
+  trace,
+  setSpanContext,
+  SpanKind,
+} from '@opentelemetry/api';
+
+import PKG = require('../package.json');
```

Comment:
I have updated it but looking forward hearing from @feywind

Reply:
The tl;dr is that I'd use `import` as long as it works.

Here is some info on that: https://stackoverflow.com/questions/52534910/difference-between-import-x-requirex-and-const-x-requirex-in-typ

It more or less matches my guess, which is that `import` pulls in types, while `const` doesn't. Both have the same effect at runtime.

Some of the code in this package is really old, and the `import` syntax might not have existed back then, so that's probably why the other module has it.

Reply:
Do you want me to change it?